src/app.ts

```diff
@@ -2,36 +2,46 @@ import express from "express";
 import apicache from "apicache";
 import cors from "cors";
 import cron from "node-cron";
+import path from "path";
+import fs from "fs";
 import { getStatus } from "./get-status";
 import { getBudgets } from "./get-budgets";
 import { log } from "./debug";
 import getCarouselData from "./get-carousel-data";
 import getPrice from "./get-price";
-import path from "path";
-import fs from "fs";
+import getCirculatingSupply from "./get-circulating-supply";
 
 const app = express();
 const cache = apicache.middleware;
 const port = process.env.PORT || 8081;
 const CAROUSEL_DATA_PATH = path.join(__dirname, "../carousel-data.json");
+const CIRCULATING_SUPPLY_DATA_PATH = path.join(__dirname, "../circulating-supply-data.json");
 
 app.use(cors());
 app.use(express.json());
 
-const scheduleCronJob = () => {
+const scheduleCronJob = async () => {
   console.log("Scheduling cron job...");
 
-  const fetchAndWriteData = async () => {
+  const fetchAndWriteCarouselData = async () => {
     const carouselData = await getCarouselData();
 
     fs.writeFileSync(CAROUSEL_DATA_PATH, JSON.stringify(carouselData, null, 2));
   };
 
+  const fetchAndWriteCirculatingSupplyData = async () => {
+    const circulatingSupplyData = await getCirculatingSupply();
+
+    fs.writeFileSync(CIRCULATING_SUPPLY_DATA_PATH, JSON.stringify(circulatingSupplyData, null, 2));
+  };
+
   // Fetch data initially such that we have something to serve. There will at most
   // be a buffer of 5 minutes from this running until the first cron execution.
-  fetchAndWriteData();
+  await fetchAndWriteCirculatingSupplyData();
+  await fetchAndWriteCarouselData();
 
-  cron.schedule("*/5 * * * *", async () => fetchAndWriteData());
+  cron.schedule("*/5 * * * *", async () => await fetchAndWriteCarouselData());
+  cron.schedule("* * * * *", async () => await fetchAndWriteCirculatingSupplyData());
```

Comment:
The schedule looks a bit aggressive (every minute). Since the query can be quite slow and expensive as it is fetching all system account locks.
Also can be simplified to.
```suggestion
  cron.schedule("*/5 * * * *", fetchAndWriteCirculatingSupplyData);
```